### PR TITLE
Refactoring #experience .splash selector blocks. closes #1726

### LINKED ISSUE #1726: Refactor #experience .splash selector and similar blocks
- [x] `#experience .splash` around line 256
- [x] `#experience.Designing .splash` around line 991
- [x] `#experience.When .splash` around line 999

Blocks #1680

### PR DESCRIPTION
**Pull request by webinista**
- Changing #content .splash h2 to use a .va-headingflag selector
- Changing #content .splash h2.tagline to .va-headingflag--tagline

### CODE REVIEW

File: _sass/_va.scss [hunk 1]
```diff
@@ -991,49 +989,44 @@ span.next {
...
-      padding: .5em .25em; 
+  a {
+    margin: 0 !important;
+    background: $color-gold !important; 
+    border-bottom: none !important;
+    color: $color-primary-darkest !important;
+    font-size: .9em !important;
+    line-height: 1.4em !important;
+    padding: .5em .25em; 
+    text-decoration: none;
```
Comment (emilyville) [inline _sass/_va.scss]: I think this needs to be in the h2 in order to be applied?
screen shot 2016-04-05 at 2 18 25 pm

**Comment by emilyville** [comment]
Back at ya - it looks fine on all the post pages, just the landing page is looking off.

**Comment by webinista** [comment]
Finally tracked down where to make that change, @emilyville. Reassigning to you.

**Comment by emilyville** [comment]
LGTM!

**Pull request merged**